app/_data/tables/plugin_index.yml

```diff
@@ -469,8 +463,8 @@
       free: No
       plus: No
```

Comment:
Based on this doc https://docs.google.com/spreadsheets/d/1DLtOGCFL_eQAUDhvucYaJ6LBFwSSm3LMEgg6032-LNM/edit#gid=1122796130 we set Response Transformer to available in free, plus and ent. Looks like this doesn't match the pricing sheet

Reply:
Unless I'm misreading the way this content is formatted.

Reply:
I think you might be looking at Response Transformer Advanced. It's harder to tell with the way github shows the preview, but here's the full file+those lines: https://github.com/Kong/docs.konghq.com/blob/d1a4f095aab3648c27d1df36faef4c6a9aa03a14/app/_data/tables/plugin_index.yml#L462

Reply:
Great, I thought that may be the case but just wanted to confirm.